README.md

```diff
@@ -164,4 +166,4 @@ Released under the [MIT license](LICENSE).
 [soda]: https://github.com/buymeasoda/soda-theme
 [ba]: https://github.com/jonschlinkert/sublime-monokai-extended/issues/4
 [yfm]: http://assemble.io/docs/YAML-front-matter.html "YAML Front Matter"
-[gfm]: https://help.github.com/articles/github-flavored-markdown#syntax-highlighting "GitHub Flavored Markdown"
\ No newline at end of file
+[gfm]: https://help.github.com/articles/github-flavored-markdown#syntax-highlighting "GitHub Flavored Markdown"
```

Comment:
I don't know why it looks like I'm changing this line, but I'm not. See for yourself:

https://raw.githubusercontent.com/jonschlinkert/sublime-monokai-extended/master/README.md

https://raw.githubusercontent.com/arielelkin/sublime-monokai-extended/01c74f92d28af3e8ac0f08cc9387c3a0a7fcb75f/README.md